src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -423,14 +517,40 @@ private async Task HandleErrors(string documentId, Exception e)
             await ThrowOnUnavailableStream(documentId, e).ConfigureAwait(false);
         }
 
-        private IDisposable ConcurrencyCheck()
+        private IDisposable _streamLockDisposable;
```

Comment:
you have a warning here, always good to review your PR few min after GH actions are complete to check for those

Reply:
done